__tests__/app.test.js

```diff
@@ -0,0 +1,58 @@
+const request = require("supertest");
+const db = require("../db/connection");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+
+afterAll(() => {
+  return db.end();
+});
+
+beforeEach(() => {
+  return seed(data);
+});
+
+describe("/api/healthcheck: HealthCheck to confirm connection with sever", () => {
```

Comment:
This test is extra to the ticket and hasn't been asked for. Be careful when completing tickets to only do the work asked for within the specific ticket